cardano-crypto-class/src/Cardano/Crypto/VRF/Simple.hs

```diff
@@ -52,7 +49,7 @@ q :: Integer
 q = C.ecc_n $ C.common_curve curve
 
 newtype Point = ThunkyPoint C.Point
-  deriving (Eq, Generic)
+  deriving stock (Eq, Generic)
```

Comment:
Not bad, but unrelated.

If we wanted to do this consistently, we should do it consistently in a separate patch (and perhaps separate PR).

Reply:
Was just trying to shut up the warnings. But should be done separately, yes.